src/moscot/problems/cross_modality/_mixins.py

```diff
@@ -183,6 +187,60 @@ def cell_transition(  # type: ignore[misc]
             key_added=key_added,
         )
 
+    def annotation_mapping(  # type: ignore[misc]
+        self: CrossModalityTranslationMixinProtocol[K, B],
+        mapping_mode: Literal["sum", "max"],
+        annotation_label: str,
+        forward: bool,
+        source: str = "src",
+        target: str = "tgt",
+        scale_by_marginals: bool = True,
+        other_adata: Optional[str] = None,
+        cell_transition_kwargs: Mapping[str, Any] = types.MappingProxyType({}),
+    ) -> pd.DataFrame:
+        """Transfer annotations between distributions.
+
+        This function transfers annotation labels (e.g. cell types) between groups of cells.
+
+        Parameters
+        ----------
+        mapping_mode
+            How to decide which label to transfer. Valid options are:
+
+            - ``'max'`` - pick the label of the annotated cell with the highest mapping weight.
+            - ``'sum'`` - compute :meth:`cell_transition` of annotation labels to target cells and
+              pick the label with the highest transition weight.
+        annotation_label
+            Key in :attr:`~anndata.AnnData.obs` where the annotation is stored.
+        forward
+            If :obj:`True`, transfer the annotations from ``source`` to ``target``.
+        source
+            Key identifying the source distribution.
+        target
+            Key identifying the target distribution.
+        scale_by_marginals
+            Whether to scale by the source :term:`marginals`.
+        other_adata
```

Comment:
I don't think the user specifies this. also see below, it is inferred. hence remove.